### ISSUE TITLE
In BART pre training do we replace multiple [MASK] with single one?

### ISSUE THREAD

**Issue by saksham-singhal** [pytorch/fairseq#1603]
Hello,

In BART pretraining, while creating input sequence, we end up with something like the following input sequence:

 [BOS] a b [MASK] [MASK] e 

do we replace it with

[BOS] a b [MASK] e

Basically do we replace multiple consecutive "[MASK]" tokens with a single one?

@ngoyal2707

**Comment by lematt1991** [comment]
No, I don't believe so.  CC @ngoyal2707 for confirmation
